snippets/bt-ll-sw-split/bt-ll-sw-split.overlay

```diff
@@ -0,0 +1,13 @@
+&bt_hci_controller {
+	status = "okay";
+};
+
+&bt_hci_sdc {
+	status = "disabled";
+};
```

Comment:
Not sure what do with other device tree nodes. Maybe simply remove it from here? But then there are two nodes enabled which may be a bit confusing

Reply:
out of tree defined nodes can't be here, that for sure

Reply:
Discussed offline. Will remove that part and keep them both enabled. 

Follow up work would include removing controller selection. Need to think though how to handle that nicely when there are two nodes enabled